app/components/home/Home.js

```diff
@@ -1,107 +1,184 @@
 // import redux, react
 import React from 'react'
-import { connect } from 'react-redux';
 
 // Material theme
 import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
-import {Paper} from 'material-ui';
+import { SvgIcon } from 'material-ui';
+import { textLight } from '../colors'
 
-//External Components Modular
+// Custom components
 import Signin from './Signin'
 
-//About us
-const defaultPix = '../../../src/images/defaultProfile.png';
-
+// Page content
 const navbarLinks = {
   about: 'About Us',
   contact: 'Contact'
 }
+const teamMembers = [
+  {
+    name: 'Emily Ho',
+    img: '../../../src/images/coders/emily_ho.jpeg',
```

Comment:
Have that thing where I can't see the photos despite them being in the right place and correctly sourced. Is this that 'require' complexity again?